client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -128,6 +128,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 	const siteTitle = site?.name;
 	const siteDescription = site?.description;
 	const { shouldLimitGlobalStyles } = useSiteGlobalStylesStatus( site?.ID );
+	const { data: siteActiveTheme } = useActiveThemeQuery( site?.ID ?? 0, !! site?.ID );
```

Comment:
Non-blocking nit: useActiveThemeQuery function already checks if the site id is not a falsy value. If it is, it won't make the request. We can simply pass `useActiveThemeQuery( site?.ID`)` and it would be the same.

Reply:
We need to provide a number, the site id value is not nullable.

Reply:
Fair enough; we can leave the first arg as 0. The second one is not needed since !!0 is false and it short circuits the query. Let's leave it as it is since it's not worth updating it.